ros/src/computing/perception/localization/packages/autoware_connector/nodes/can_odometry/can_odometry_core.h

```diff
@@ -39,7 +39,7 @@
 #include <tf/transform_broadcaster.h>
 
 // User Defined Includes
-#include "autoware_msgs/CanInfo.h"
+#include "autoware_can_msgs/CANInfo.h"
```

Comment:
@funaoka Then in that case this would be wrong.

Reply:
@amc-nu I think 3rd choice is not a good idea. 1st and 2nd choices are the fastest way.

1. Merge this pull request as is, and then fix name in another pull request
1. Merge this pull request with fixing name
1. Revert the fixing name, and then fix name in another pull request

Reply:
Both are good, I think :)

Depending on esteve?

Reply:
I'd rather merge this as is. There are other naming issues that need to be addressed as well (e.g. Ros vs ROS), but I think they're out of scope here. I changed `CanInfo` because there were already two other messages with a different naming convention (`CANData` and `CANPacket`).